### PR TITLE
Adding rhel and rchos hybrid worker installer

### PR DESCRIPTION
**Pull request by paigerube14**
Want to be able to install a hybrid cluster of both rchos and rhel nodes

https://issues.redhat.com/browse/PERFSCALE-2727

### CODE REVIEW

**Comment by paigerube14** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-aws-4.15-nightly-x86-hybrid-control-plane-3nodes

**Comment by paigerube14** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-aws-4.15-nightly-x86-hybrid-control-plane-3nodes

**Comment by paigerube14** [comment]
https://github.com/openshift/release/pull/47579 added scaling for rhel nodes, will use this to scale

**Comment by paigerube14** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-aws-4.15-nightly-x86-hybrid-control-plane-12nodes

**Comment by paigerube14** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-aws-4.15-nightly-x86-hybrid-control-plane-12nodes

**Comment by paigerube14** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-aws-4.15-nightly-x86-hybrid-control-plane-12nodes

**Comment by paigerube14** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-aws-4.15-nightly-x86-hybrid-control-plane-12nodes

**Comment by paigerube14** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-aws-4.15-nightly-x86-hybrid-control-plane-12nodes

**Comment by paigerube14** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-aws-4.15-nightly-x86-hybrid-control-plane-12nodes

**Comment by paigerube14** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-aws-4.15-nightly-x86-hybrid-control-plane-3nodes

**Comment by paigerube14** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-aws-4.15-nightly-x86-hybrid-control-plane-12nodes

**Comment by paigerube14** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-aws-4.15-nightly-x86-hybrid-control-plane-12nodes

**Comment by paigerube14** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-aws-4.15-nightly-x86-hybrid-control-plane-12nodes

File: ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aws-4.15-nightly-x86.yaml [hunk 1]
```diff
@@ -68,8 +72,22 @@ tests:
     - chain: openshift-qe-control-plane
     workflow: openshift-qe-installer-aws
   timeout: 2h30m0s
-- always_run: false
-  as: conc-builds-3nodes
+- as: hybrid-control-plane-12nodes
```
Comment (qiliRedHat) [inline ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aws-4.15-nightly-x86.yaml:75]: @paigerube14 Why 12 nodes are used? Is there a limitation on the node number?

File: ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aws-4.15-nightly-x86.yaml [hunk 2]
```diff
@@ -68,8 +72,22 @@ tests:
...
+  cron: 0 10 * * 3
```
Comment (qiliRedHat) [inline ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aws-4.15-nightly-x86.yaml:76]: @paigerube14 Regarding the cron, this profile(120 nodes) we did not run them regularly in Jenkins before, it used to be run manually once per release. If we don't have the need to escalate the priority(frequency) to run this profile, what do you think  to make it always_run: false?

File: ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aws-4.15-nightly-x86.yaml (hunk 1 above)
Comment (paigerube14) [reply ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aws-4.15-nightly-x86.yaml:75]: no limitation, was trying to pick a lower number that we could see the rhel and rchos nodes

**Comment by paigerube14** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-aws-4.15-nightly-x86-hybrid-control-plane-12nodes

**Comment by paigerube14** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-aws-4.15-nightly-x86-hybrid-control-plane-12nodes

**Comment by paigerube14** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-aws-4.15-nightly-x86-hybrid-control-plane-12nodes

**Comment by paigerube14** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-aws-4.15-nightly-x86-hybrid-control-plane-12nodes

**Comment by paigerube14** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-aws-4.15-nightly-x86-hybrid-control-plane-12nodes

**Comment by paigerube14** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-aws-4.15-nightly-x86-hybrid-control-plane-12nodes

**Comment by paigerube14** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-aws-4.15-nightly-x86-hybrid-control-plane-12nodes

**Comment by paigerube14** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-aws-4.15-nightly-x86-hybrid-control-plane-12nodes

**Comment by paigerube14** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-aws-4.15-nightly-x86-hybrid-control-plane-12nodes

**Comment by paigerube14** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-aws-4.15-nightly-x86-hybrid-control-plane-12nodes

**Comment by paigerube14** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-aws-4.15-nightly-x86-hybrid-control-plane-12nodes

**Comment by paigerube14** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-aws-4.15-nightly-x86-hybrid-control-plane-12nodes

**Comment by paigerube14** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-aws-4.15-nightly-x86-hybrid-control-plane-12nodes

**Comment by paigerube14** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-aws-4.15-nightly-x86-hybrid-control-plane-12nodes

**Comment by paigerube14** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-aws-4.15-nightly-x86-hybrid-control-plane-12nodes

**Comment by paigerube14** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-aws-4.15-nightly-x86-hybrid-control-plane-12nodes

**Comment by paigerube14** [comment]
/pj-rehearse pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-aws-4.15-nightly-x86-hybrid-control-plane-12nodes

**Comment by paigerube14** [comment]
/pj-rehearse pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-aws-4.15-nightly-x86-hybrid-control-plane-12nodes

**Comment by paigerube14** [comment]
/pj-rehearse pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-aws-4.15-nightly-x86-hybrid-control-plane-120nodes

**Comment by paigerube14** [comment]
/pj-rehearse pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-aws-4.15-nightly-x86-hybrid-control-plane-120nodes

**Comment by paigerube14** [comment]
/pj-rehearse pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-aws-4.15-nightly-x86-hybrid-control-plane-120nodes

**Comment by paigerube14** [comment]
/pj-rehearse pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-aws-4.15-nightly-x86-hybrid-control-plane-120nodes

**Comment by paigerube14** [comment]
/pj-rehearse pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-aws-4.15-nightly-x86-hybrid-control-plane-120nodes

**Comment by paigerube14** [comment]
/pj-rehearse pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-aws-4.15-nightly-x86-hybrid-control-plane-120nodes

**Comment by paigerube14** [comment]
/pj-rehearse pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-aws-4.15-nightly-x86-hybrid-control-plane-120nodes

**Comment by paigerube14** [comment]
/pj-rehearse pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-aws-4.15-nightly-x86-hybrid-control-plane-120nodes

**Comment by paigerube14** [comment]
/pj-rehearse pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-aws-4.15-nightly-x86-hybrid-control-plane-120nodes

**Comment by paigerube14** [comment]
/pj-rehearse pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-aws-4.15-nightly-x86-hybrid-control-plane-120nodes

**Comment by paigerube14** [comment]
/pj-rehearse pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-aws-4.15-nightly-x86-hybrid-control-plane-120nodes

**Comment by paigerube14** [comment]
/pj-rehearse pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-aws-4.15-nightly-x86-hybrid-control-plane-120nodes

**Comment by paigerube14** [comment]
/pj-rehearse pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-aws-4.15-nightly-x86-hybrid-control-plane-120nodes

**Comment by paigerube14** [comment]
@qiliRedHat PTAL

**Comment by paigerube14** [comment]
@skordas @svetsa-rh PTAL, will help with the hybrid 4.16 testing
passing job here: https://prow.ci.openshift.org/view/gs/test-platform-results/pr-logs/pull/openshift_release/46695/rehearse-46695-pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-aws-4.15-nightly-x86-hybrid-control-plane-120nodes/1785381047475638272

**Review by skordas — APPROVED** [review]
/lgtm

**Pull request merged**